content/guides/references/release-stages.md

```diff
@@ -0,0 +1,111 @@
+    semver is bumped by a major version
+  - In some cases early access to these changes are possible through public
+    packages
+- Removals
+  - Removals will only be introduced during major releases in which semver is
+    bumped by a major version
+  - In the majority of cases, removals will be proceeded by a period in which
+    the feature is ‘deprecated’
+  - Migration steps will be provided to minimize pain points in these situations
+
+## Beta
+
+Beta features consist of features that we are looking to gather user feedback on
+by providing them as default functionality before an official release. Beta
+features may not be feature complete, but they should be close enough where
+large, possibly breaking, changes to, either the API or functionality are few and far between.
+
+**Strategies for releasing and communicating changes to Beta features**
+
+- Modifications
+  - Modifications can be introduced during any release cycle.
+  - Affected users will be pointed to documentation if there blockers are
+    perceived to be possible.
+- Breaking Changes
+  - Breaking changes can be introduced during any release cycle and do not bump
+    semver by a major version
+  - Breaking changes require in-app messaging to help users unblock themselves
+    as needed
+  - The introduction and impact of breaking changes should be carefully considered
+    at this stage as the cost to users is higher than with an alpha feature.
+- Removals
+  - Removal of any new functionality related to the feature can happen during
+    any release cycle and will increment semver. These do not need to coincide
+    with a major release that is bumping semver by a major version.
+  - Removals at this stage will be rare and a migration strategy along with
+    reasoning for the removal will be documented.
+
+## Alpha
+
+Alpha features consist of features that we are looking to gather user feedback
+on by providing them as default functionality before an official release. Alpha
+features may not be feature complete and changes are possible during this stage.
+
+**Strategies for releasing and communicating changes to Alpha features**
+
+- Modifications
+  - Modifications can be introduced during any release cycle and may or may not
+    increment semver.
+  - Affected users will be pointed to documentation if there blockers are
+    perceived to be possible.
+- Breaking Changes
+  - Breaking changes can be introduced during any release cycle and do not bump
+    semver by a major version
+- Removals
+  - Removal of any new functionality related to the feature can happen during
+    any release cycle and does not bump semver by a major version.
+
+## Experimental
+
+Experimental features consist of features that we are looking to gather user
+feedback on before we commit fully to further development of the feature.
+These type of features require you to opt-in to via a configuration
+value before the feature is activated. These types of features may be unstable
```

Comment:
I'm still not sold on calling out an experiment is `opt-in to via a configuration` because some "alpha/beta" ready concepts will need an opt'd-in to  on the end-to-end side and won't necessarily be able to provided by default.

Reply:
Would it be better this way:

> These type of features require you to opt-in to via an `experimental`-prefixed configuration value

I think that came up in one of our chat about this. 

Agree that not all features (even fully released features) can be on by default at the same time. I think what's more important is that users can't accidentally turn on an experiment without knowing that it's an experiment.

Reply:
```suggestion
These type of features require you to opt-in to via a configuration
value prefixed with `experimental`. These types of features may be unstable
```